zk_toolbox/crates/zk_inception/src/commands/prover/args/run.rs

```diff
@@ -76,8 +187,14 @@ impl WitnessVectorGeneratorArgs {
     }
 }
 
+#[derive(Debug, Clone, Parser, Default)]
+pub struct FriProverRunArgs {
+    #[clap(long)]
+    pub max_allocation: Option<usize>,
```

Comment:
/// Memory allocation limit in bytes (for prover component)
    #[clap(long)]
    pub max_allocation: Option<usize>,